soccer/NewRefereeModule.cpp

```diff
@@ -106,11 +106,11 @@ static const int KickVerifyTime_ms = 250;
 // the ref halts/stops, make this false
 static const bool CancelBallPlaceOnHalt = true;
 
-NewRefereeModule::NewRefereeModule(SystemState& state)
+NewRefereeModule::NewRefereeModule(Context* context)
```

Comment:
Kind of curious about the decision to move from references to pointers.

Reply:
I have a vendetta against references. 

In all seriousness, I am not a fan of references that get mutated or stored outside of the scope of the function as parameters to that function. I prefer pointers in this case because it makes the semantics more explicit at the callsite.